packages/amazonq/.changes/next-release/Bug Fix-9cebf072-0ebd-476d-a1bf-babd27ef9cc5.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Amazon Q /dev: fix for stop button showing for Gumby"
```

Comment:
Can you call it `Code Transformation` instead?

Reply:
Yes, one sec